src/test/java/com/epam/prejap/teatrees/block/TBlockTest.java

```diff
@@ -0,0 +1,87 @@
+package com.epam.prejap.teatrees.block;
+
+import com.epam.prejap.teatrees.TeaTrees;
+import com.epam.prejap.teatrees.game.Playfield;
+import com.epam.prejap.teatrees.game.Printer;
+import com.epam.prejap.teatrees.game.Waiter;
+import com.epam.prejap.teatrees.player.Player;
+import com.epam.prejap.teatrees.player.RandomPlayer;
+import org.testng.annotations.AfterMethod;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+
+import java.io.ByteArrayOutputStream;
+import java.io.PrintStream;
+
+import static org.testng.Assert.assertEquals;
+
+@Test
+public class TBlockTest {
+
+    private TBlock tBlock;
+    private ByteArrayOutputStream outputStream;
+    TeaTrees teatrees;
+    BlockFeed blockFeed;
+    Printer printer;
+    Playfield playfield;
+    Waiter waiter;
+    Player player;
+
+    @BeforeMethod
+    private void setup() {
+        tBlock = new TBlock();
+        outputStream = new ByteArrayOutputStream();
+        System.setOut(new PrintStream(outputStream));
+        blockFeed = new BlockFeed();
+        printer = new Printer(System.out);
```

Comment:
Why reset the global variable `System.out` with a custom output stream, if we can pass this custom output stream to the Printer directly?

Reply:
fixed